### ISSUE TITLE
Update package description with language information

### ISSUE THREAD

**Issue by quinnanya** [mjockers/syuzhet#38]
Would it be possible to update the Cran description (and/or GitHub readme) to include the languages and/or scripts that Syuzhet supports? Unlike when #29 was posted, NRC has support for multiple non-Latin scripts, but there's a mismatch between what it supports and the list of languages that throw errors when you try to use then with NRC in Syuzhet. Having the information about what languages are compatible upfront would be really helpful for managing expectations. (Happy to suggest some prose, including mitigation strategies for improving results for inflected languages, if it'd help.)

**Comment by mjockers** [comment]
Yes, please do suggest!  I have not looked at NRC in ages, so I suspect that Syuzhet does not have the current version.
